src/scripts/medgen_conflicts_removals.py

```diff
@@ -0,0 +1,146 @@
+"""MedGen conflicts: Removals
```

Comment:
## Only removing xrefs on specific Mondo IDs
@matentzn Per your instruction, I followed Megan's original directive and removed xrefs w/ on specific Mondo IDs.

I re-ran this diff analysis, and it looks good. Only xrefs are removed, and unlike before, any xrefs removed are only being removed on 1 term.


|metric_name                               |val |
|------------------------------------------|----|
|n_lines_removed                           |2669|
|n_xrefs_removed                           |2669|
|n_non_xrefs_removals                      |0   |
|n_medgen_removals                         |0   |
|n_umls_removals                           |2669|
|n_single_cui_removed_from_xrefs_n_times__1|2669|

Reply:
great, i will take it from here! Thank you!